symfony/src/Component/Model/EnableInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Component\Model;
+
+/**
+ * Entities implementing this interface keep a state of enabled. Must use with User Profiles!
+ */
+interface EnableInterface
```

Comment:
Deze aanpassingen dan ook moeten doorvoeren in huidige registration persister + profiel